stream/__init__.py

```diff
@@ -5,7 +5,7 @@
 __copyright__ = "Copyright 2014, Stream.io, Inc"
 __credits__ = ["Thierry Schellenbach, mellowmorning.com, @tschellenbach"]
 __license__ = "BSD-3-Clause"
-__version__ = "5.1.1"
+__version__ = "6.0.0"
```

Comment:
no need to change this, we'll bump it when we release it

Reply:
fixed